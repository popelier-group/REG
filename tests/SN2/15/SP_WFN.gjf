# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.90605000   -0.00005200    0.00020200
 H                 -0.88894700   -0.91019700   -0.56707600
 H                 -0.88893700    0.94631400   -0.50433500
 H                 -0.88895600   -0.03627700    1.07205700
 Br                 1.62022600    0.00000500   -0.00001900
 Cl                -3.26877300    0.00001200   -0.00004500

frame.wfn
