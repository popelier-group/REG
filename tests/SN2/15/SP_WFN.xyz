
6

C        0.000108    0.008237    0.791837
H       -0.557615   -0.907923    0.784257
H       -0.514302    0.949041    0.764823
H        1.072285   -0.016948    0.775125
Br        0.000166   -0.018140   -1.734301
Cl       -0.000402    0.033018    3.154430
