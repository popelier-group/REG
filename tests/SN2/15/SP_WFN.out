 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.15/Gau-192522.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.15/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe PID=    192523.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.90605  -0.00005   0.0002 
 H                    -0.88895  -0.9102   -0.56708 
 H                    -0.88894   0.94631  -0.50434 
 H                    -0.88896  -0.03628   1.07206 
 Br                    1.62023   0.00001  -0.00002 
 Cl                   -3.26877   0.00001  -0.00005 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.906050   -0.000052    0.000202
      2          1           0       -0.888947   -0.910197   -0.567076
      3          1           0       -0.888937    0.946314   -0.504335
      4          1           0       -0.888956   -0.036277    1.072057
      5         35           0        1.620226    0.000005   -0.000019
      6         17           0       -3.268773    0.000012   -0.000045
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.072595   0.000000
     3  H    1.072595   1.857571   0.000000
     4  H    1.072603   1.857550   1.857551   0.000000
     5  Br   2.526276   2.728731   2.728687   2.728856   0.000000
     6  Cl   2.362723   2.610283   2.610252   2.610412   4.888999
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000108    0.008237    0.791837
      2          1           0       -0.557615   -0.907923    0.784257
      3          1           0       -0.514302    0.949041    0.764823
      4          1           0        1.072285   -0.016948    0.775125
      5         35           0        0.000166   -0.018140   -1.734301
      6         17           0       -0.000402    0.033018    3.154430
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         145.3273758           0.8496792           0.8496791
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       171.6715775510 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  6.15D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147513.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 EnCoef did   100 forward-backward iterations
 SCF Done:  E(RB3LYP) =  -3074.38731728     A.U. after   12 cycles
            NFock= 12  Conv=0.44D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.61154-101.32637 -62.26460 -56.08332 -56.08185
 Alpha  occ. eigenvalues --  -56.08185 -10.09152  -9.23950  -8.47348  -7.00065
 Alpha  occ. eigenvalues --   -6.99626  -6.99626  -6.29951  -6.29466  -6.29466
 Alpha  occ. eigenvalues --   -2.41129  -2.40997  -2.40997  -2.40614  -2.40614
 Alpha  occ. eigenvalues --   -0.60463  -0.56349  -0.52231  -0.30911  -0.30910
 Alpha  occ. eigenvalues --   -0.17608  -0.11603  -0.11603  -0.09934  -0.09934
 Alpha  occ. eigenvalues --   -0.09482
 Alpha virt. eigenvalues --    0.09351   0.11510   0.15147   0.15147   0.16013
 Alpha virt. eigenvalues --    0.17665   0.18651   0.18651   0.20419   0.21498
 Alpha virt. eigenvalues --    0.21498   0.22993   0.22994   0.24693   0.30072
 Alpha virt. eigenvalues --    0.31670   0.31671   0.36787   0.39016   0.54292
 Alpha virt. eigenvalues --    0.54739   0.54739   0.57663   0.57664   0.59265
 Alpha virt. eigenvalues --    0.63773   0.63774   0.69514   0.77089   0.77090
 Alpha virt. eigenvalues --    0.77692   0.83512   0.83513   0.84082   0.84083
 Alpha virt. eigenvalues --    0.84419   0.90167   1.08224   1.08224   1.08782
 Alpha virt. eigenvalues --    1.08783   1.11996   1.17748   1.22697   1.22698
 Alpha virt. eigenvalues --    1.32540   1.52410   1.61784   1.61785   1.67862
 Alpha virt. eigenvalues --    1.95748   2.05318   2.05319   2.07717   2.07719
 Alpha virt. eigenvalues --    2.19694   2.19694   2.31390   2.46731   2.46732
 Alpha virt. eigenvalues --    2.55387   2.61586   2.61587   2.69223   2.77713
 Alpha virt. eigenvalues --    2.77714   2.98606   3.21585   3.86702   3.86702
 Alpha virt. eigenvalues --    4.34624   4.34625   4.36289   4.36289   4.54723
 Alpha virt. eigenvalues --    6.75770   7.75400   7.75402   7.83196   9.86096
 Alpha virt. eigenvalues --   24.03880  26.02994  26.02995  26.26334  48.06144
 Alpha virt. eigenvalues --  215.83751 289.95934 289.95934 290.036821020.82960
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.081157   0.420659   0.420664   0.420643   0.029900  -0.044974
     2  H    0.420659   0.512762  -0.032439  -0.032451  -0.030283  -0.022118
     3  H    0.420664  -0.032439   0.512761  -0.032449  -0.030288  -0.022120
     4  H    0.420643  -0.032451  -0.032449   0.512768  -0.030261  -0.022110
     5  Br   0.029900  -0.030283  -0.030288  -0.030261  35.715411   0.013180
     6  Cl  -0.044974  -0.022118  -0.022120  -0.022110   0.013180  17.654036
 Mulliken charges:
               1
     1  C   -0.328049
     2  H    0.183869
     3  H    0.183872
     4  H    0.183860
     5  Br  -0.667659
     6  Cl  -0.555893
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.223552
     5  Br  -0.667659
     6  Cl  -0.555893
 Electronic spatial extent (au):  <R**2>=           1141.7039
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0008    Y=             -0.0307    Z=             -2.9217  Tot=              2.9219
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -43.8725   YY=            -43.8770   ZZ=            -86.2969
   XY=              0.0001   XZ=              0.0054   YZ=             -0.4441
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             14.1429   YY=             14.1385   ZZ=            -28.2814
   XY=              0.0001   XZ=              0.0054   YZ=             -0.4441
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              1.1135  YYY=             -0.4025  ZZZ=           -133.6977  XYY=             -1.1060
  XXY=             -0.2377  XXZ=            -15.2896  XZZ=              0.0120  YZZ=             -1.0791
  YYZ=            -15.3010  XYZ=              0.0117
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -71.4495 YYYY=            -71.5969 ZZZZ=          -1732.1549 XXXY=              0.0089
 XXXZ=              0.8627 YYYX=             -0.0237 YYYZ=             -7.0864 ZZZX=              0.1127
 ZZZY=            -10.2364 XXYY=            -23.8430 XXZZ=           -251.3030 YYZZ=           -251.3629
 XXYZ=             -2.4354 YYXZ=             -0.7589 ZZXY=              0.0171
 N-N= 1.716715775510D+02 E-N=-7.696124746644D+03  KE= 3.069965311156D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE764\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\ZZALSPA2\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-0.90605,-0.000052,0.000202\H,0,-0.888947,-0.910197,-0.567076\H,0,
 -0.888937,0.946314,-0.504335\H,0,-0.888956,-0.036277,1.072057\Br,0,1.6
 20226,0.000005,-0.000019\Cl,0,-3.268773,0.000012,-0.000045\\Version=ES
 64L-G16RevC.01\State=1-A\HF=-3074.3873173\RMSD=4.382e-09\Dipole=1.1495
 603,-0.0000478,0.0001922\Quadrupole=-21.0299855,10.5150547,10.5149308,
 0.0001371,-0.0004976,-0.0000017\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 ART, GLORY, FREEDOM FAIL, BUT NATURE STILL IS FAIR.

                                           -- BYRON
 Job cpu time:       0 days  0 hours  0 minutes 15.2 seconds.
 Elapsed time:       0 days  0 hours  0 minutes  8.1 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:47:17 2020.
